app/models/exchange_rate_currency_rate.rb

```diff
@@ -4,6 +4,9 @@ class ExchangeRateCurrencyRate < Sequel::Model
   RATES_FILE = 'data/exchange_rates/all_rates.csv'.freeze
   SPOT_RATES_FILE = 'data/exchange_rates/all_spot_rates.csv'.freeze
 
+  many_to_one :exchange_rate_currency, key: :currency_code, primary_key: :currency_code, class_name: ExchangeRateCurrency
```

Comment:
You really ought not to need the class name now, this is worked out with the equivalent of inflections